types/broccoli-plugin/index.d.ts

```diff
@@ -0,0 +1,74 @@
+// Type definitions for broccoli-plugin 1.3
+// Project: https://github.com/broccolijs/broccoli-plugin
+// Definitions by: Dan Freeman <https://github.com/dfreeman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export type BroccoliNode = BroccoliPlugin | string;
+
+export interface BroccoliPluginOptions {
+    /**
+     * The name of this plugin. Defaults to `this.constructor.name`.
+     */
+    name?: string;
+
+    /**
+     * A descriptive annotation. Useful for debugging, to tell multiple
+     * instances of the same plugin apart.
+     */
+    annotation?: string;
+
+    /**
+     * If `true`, the output directory is not automatically emptied between
+     * builds. Defaults to `false`.
+     */
+    persistentOutput?: boolean;
+
+    /**
+     * If `true`, a cache directory is created automatically and the path is
+     * set at `cachePath`. If `false`, a cache directory is not created and
+     * `this.cachePath` is `undefined`. Defaults to `true`.
+     */
+    needsCache?: boolean;
+}
+
+export default class BroccoliPlugin {
```

Comment:
Seeing from https://github.com/broccolijs/broccoli-plugin#example-usage this needs to be `export = BroccoliPlugin` as there is no ".default" on the package. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
Ah, thanks @sheetalkamat! Just inattentiveness on my part 🙂